app/assets/stylesheets/components/_player.scss

```diff
@@ -97,7 +97,7 @@
     display:inline-block;
     float:left;
     height:16px;
-    margin:17px 0 0 20px;
+    margin:16px 0 0 20px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space